docs/concepts.md

```diff
@@ -3,7 +3,7 @@
 ## DCE
 
 _Disposable Cloud Environment (DCE)_ provide temporary, limited access to Amazon Web 
-Services (AWS) accounts. Administrators can configure this limited access to expire based on time or budget. When the access expires, DCE destroys all of the resources in the account and returns the [account to the [account pool](#account-pool)
+Services (AWS) accounts. Administrators can configure this limited access to expire based on time or budget. When the access expires, DCE destroys all of the resources in the account and returns the account to the [account pool](#account-pool).
```

Comment:
This link isn't rendering
![image](https://user-images.githubusercontent.com/1153371/71927806-f2972c80-315b-11ea-8dda-8be8574ef4e5.png)